cmake/modules/dts.cmake

```diff
@@ -181,6 +181,10 @@ endif()
 
 zephyr_file(CONF_FILES ${BOARD_EXTENSION_DIRS} DTS board_extension_dts_files)
 
+foreach(dts_file ${board_extension_dts_files})
+  message(STATUS "Found board extension devicetree overlay: ${dts_file}")
```

Comment:
The same overlays are also printed down here (with no indication of origin):
https://github.com/zephyrproject-rtos/zephyr/blob/b7b3449f482cd9fb736e3a79363d390fd648051d/cmake/modules/dts.cmake#L210-L219

If you're interested in more verbosity, then this loop should be reworked to avoid logging overlays twice.

Reply:
You are right. I changed last minute when I was preparing this PR. I'll change this back to printing board extension dirs only.